### ISSUE TITLE
Improper access to nested list

### ISSUE THREAD

**Issue by amenezes** [amenezes/config-client#47]
### Platform

Other

### Steps to Reproduce

version: <= 1.0.2

First start local development container with:

```bash
docker-compose up -d
```

Second, access/show the config:

```bash
python -m config client simpleweb000 --all
```

### Expected Result

```bash
╭────────────────────────── report for filter: 'all' ──────────────────────────╮
│ {                                                                            │
│     "example": [                                                             │
│         1,                                                                   │
│         2                                                                    │
│     ],                                                                       │
│     "examples": {                                                            │
│         "float": [                                                           │
│             1.1,                                                             │
│             2.2,                                                             │
│             3.3,                                                             │
│             4.4                                                              │
│         ],                                                                   │
│         "int": [                                                             │
│             1,                                                               │
│             2                                                                │
│         ],                                                                   │
│         "str": [                                                             │
│             "example 1",                                                     │
│             "example 2",                                                     │
│             "example 3"                                                      │
│         ]                                                                    │
│     },                                                                       │
│     "first": {                                                               │
│         "second_1": [                                                        │
│             1,                                                               │
│             2                                                                │
│         ],                                                                   │
│         "second_2": [                                                        │
│             1                                                                │
│         ],                                                                   │
│         "second_3": {                                                        │
│             "third_1": [                                                     │
│                 1,                                                           │
│                 2                                                            │
│             ],                                                               │
│             "third_2": [                                                     │
│                 1,                                                           │
│                 2,                                                           │
│                 3                                                            │
│             ],                                                               │
│             "third_3": {                                                     │
│                 "fourth_1": [                                                │
│                     1,                                                       │
│                     2                                                        │
│                 ],                                                           │
│                 "fourth_2": [                                                │
│                     1                                                        │
│                 ],                                                           │
│                 "fourth_3": [                                                │
│                     1,                                                       │
│                     2,                                                       │
│                     3                                                        │
│                 ],                                                           │
│                 "fourth_4": {                                                │
│                     "fifth_1": [                                             │
│                         1,                                                   │
│                         2                                                    │
│                     ],                                                       │
│                     "fifth_2": [                                             │
│                         1,                                                   │
│                         2,                                                   │
│                         3                                                    │
│                     ]                                                        │
│                 }                                                            │
│             }                                                                │
│         }                                                                    │
│     },                                                                       │
│     "health": {                                                              │
│         "config": {                                                          │
│             "enabled": false                                                 │
│         }                                                                    │
│     },                                                                       │
│     "info": {                                                                │
│         "app": {                                                             │
│             "description": "pws simpleweb000 - development profile",         │
│             "name": "simpleweb000",                                          │
│             "password": "123"                                                │
│         }                                                                    │
│     },                                                                       │
│     "python": {                                                              │
│         "cache": {                                                           │
│             "timeout": 10,                                                   │
│             "type": "simple"                                                 │
│         }                                                                    │
│     },                                                                       │
│     "server": {                                                              │
│         "port": 8080                                                         │
│     },                                                                       │
│     "spring": {                                                              │
│         "cloud": {                                                           │
│             "consul": {                                                      │
│                 "host": "discovery",                                         │
│                 "port": 8500                                                 │
│             }                                                                │
│         }                                                                    │
│     }                                                                        │
│ }                                                                            │
╰──────────────────────────────────────────────────────────────────────────────╯

```

### Actual Result

```bash
python -m config client simpleweb000 --all

╭────────────────────────── report for filter: 'all' ──────────────────────────╮
│ {                                                                            │
│     "example": [                                                             │
│         1,                                                                   │
│         2                                                                    │
│     ],                                                                       │
│     "examples": {                                                            │
│         "float": [                                                           │
│             1.1,                                                             │
│             2.2,                                                             │
│             3.3,                                                             │
│             4.4                                                              │
│         ],                                                                   │
│         "int": [                                                             │
│             2                                                                │
│         ],                                                                   │
│         "int[0]": 1,                                                         │
│         "str": [                                                             │
│             "example 1",                                                     │
│             "example 2",                                                     │
│             "example 3"                                                      │
│         ]                                                                    │
│     },                                                                       │
│     "first": {                                                               │
│         "second_1": [                                                        │
│             2                                                                │
│         ],                                                                   │
│         "second_1[0]": 1,                                                    │
│         "second_2": [                                                        │
│             1                                                                │
│         ],                                                                   │
│         "second_3": {                                                        │
│             "third_1": [                                                     │
│                 2                                                            │
│             ],                                                               │
│             "third_1[0]": 1,                                                 │
│             "third_2": [                                                     │
│                 1,                                                           │
│                 2,                                                           │
│                 3                                                            │
│             ],                                                               │
│             "third_3": {                                                     │
│                 "fourth_1": [                                                │
│                     2                                                        │
│                 ],                                                           │
│                 "fourth_1[0]": 1,                                            │
│                 "fourth_2": [                                                │
│                     1                                                        │
│                 ],                                                           │
│                 "fourth_3": [                                                │
│                     1,                                                       │
│                     2,                                                       │
│                     3                                                        │
│                 ],                                                           │
│                 "fourth_4": {                                                │
│                     "fifth_1": [                                             │
│                         2                                                    │
│                     ],                                                       │
│                     "fifth_1[0]": 1,                                         │
│                     "fifth_2": [                                             │
│                         1,                                                   │
│                         2,                                                   │
│                         3                                                    │
│                     ]                                                        │
│                 }                                                            │
│             }                                                                │
│         }                                                                    │
│     },                                                                       │
│     "health": {                                                              │
│         "config": {                                                          │
│             "enabled": false                                                 │
│         }                                                                    │
│     },                                                                       │
│     "info": {                                                                │
│         "app": {                                                             │
│             "description": "pws simpleweb000 - development profile",         │
│             "name": "simpleweb000",                                          │
│             "password": "123"                                                │
│         }                                                                    │
│     },                                                                       │
│     "python": {                                                              │
│         "cache": {                                                           │
│             "timeout": 10,                                                   │
│             "type": "simple"                                                 │
│         }                                                                    │
│     },                                                                       │
│     "server": {                                                              │
│         "port": 8080                                                         │
│     },                                                                       │
│     "spring": {                                                              │
│         "cloud": {                                                           │
│             "consul": {                                                      │
│                 "host": "discovery",                                         │
│                 "port": 8500                                                 │
│             }                                                                │
│         }                                                                    │
│     }                                                                        │
│ }                                                                            │
╰──────────────────────────────────────────────────────────────────────────────╯
```

**Comment by amenezes** [comment]
There's a possible fix available in ther version `1.0.3a0`  that is under evaluation.

- https://pypi.org/project/config-client/1.0.3a0/

**Comment by amenezes** [comment]
Fixed in version 1.0.3
